Makefile.rules

```diff
@@ -1002,3 +1006,13 @@ doc/make_doc: $(srcdir)/doc/make_doc.in config.status
 
 gac: $(srcdir)/cnf/gac.in config.status
 	$(SHELL) ./config.status $@
+
+# TODO: This needs to be done more elegantly
```

Comment:
You probably plan to clean this up before the PR is merged. But if not, then this comment needs to be expanded, to explain what "this" is and what the TODO is.